hibernate-jipijapa/src/main/java/org/jboss/as/jpa/hibernate5/HibernateExtendedBeanManager.java

```diff
@@ -11,7 +11,7 @@
 
 import javax.enterprise.inject.spi.BeanManager;
 
-import org.hibernate.jpa.event.spi.jpa.ExtendedBeanManager;
```

Comment:
Specifically wanted to ask about this one @scottmarlow

Reply:
Looks good to me.  Seeing the org.jboss.as.test.integration.jpa.entitylistener.EntityListenersTestCase test pass, gives me more confidence about this change.